ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/plugins/ServerSentEventsTest.kt

```diff
@@ -351,28 +351,30 @@ class ServerSentEventsTest : ClientLoader(timeoutSeconds = 120) {
     }
 
     @Test
-    fun testErrorForProtocolUpgradeRequestBody() = clientTests(listOf("OkHttp")) {
-        config {
-            install(SSE)
-        }
+    fun testErrorForProtocolUpgradeRequestBody() = repeat(10000) {
```

Comment:
10k iterations seems a little excessive here 😄

Reply:
Heh, it's not finished yet 😄 will remove it after green CI tests to be sure it's not flaky anymore